### DISCUSSION TITLE
有计划支持IPQ6000系列吗？

### DISCUSSION THREAD

**Discussion by qwerttvv** [immortalwrt/immortalwrt#890] [General]
https://github.com/sdf8057/openwrt/tree/ipq6000/target/linux/ipq60xx

这里看到一个IPQ6000的移植，已经成功编译

**Comment by happyelements01** [comment]
这cpu 老老实实用 19 就行，很稳定，不用追求新功能

**Reply by zhangqimao** [reply]
19的在哪下载呀

**Comment by xlighting2017** [comment]
see here:
https://github.com/openwrt/openwrt/pull/14410

**Reply by liyiranlab** [reply]
我有台zn-m2 1g ram，成摆设了，要是可以支持就好了

**Reply by qwerttvv** [reply]
走闲鱼 我收了 @liyiranlab

**Comment by xlighting2017** [comment]
> 我有台zn-m2 1g ram，成摆设了，要是可以支持就好了

含泪50收，可走鱼

**Comment by liyiranlab** [comment]
知道为什么闲置吗？就是舍不得卖😘

**Reply by liyiranlab** [reply]
关键是要有开源维护的代码才有意义，我主要是想组网。

**Comment by qwerttvv** [comment]
自问自答一下吧
https://github.com/immortalwrt/immortalwrt/commit/23deb4ac90e16277e9d779856e77d0ed22142bc5
https://github.com/immortalwrt/immortalwrt/tree/master/target/linux/qualcommax/ipq60xx

**Reply by Glunefish** [reply]
大佬，支持ax18 znm2在线编译吗 ipq6000
